libs/database-introspection/src/mysql.rs

```diff
@@ -241,23 +241,44 @@ impl IntrospectionConnector {
             .conn
             .query_raw(sql, schema, &[schema.into(), table_name.into()])
             .expect("querying for indices");
-        let mut pk: Option<PrimaryKey> = None;
+        let mut primary_key: Option<PrimaryKey> = None;
         let indices = rows
             .into_iter()
             .filter_map(|index| {
                 debug!("Got index row: {:#?}", index);
+                let pos = index
```

Comment:
A bit hard to read, I'd prefer something like:
```
let index = ...;
let pos = index - 1;
```

Reply:
@dpetrick OK, I made a change along these lines 👍